rotkehlchen/assets/exchanges_mappings/bitfinex.py

```diff
@@ -74,4 +74,8 @@
     'ONE-2': 'ONE',
     evm_address_to_identifier('0x4922a015c4407F87432B179bb209e125432E4a2A', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'XAUT',  # noqa: E501
     evm_address_to_identifier('0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'LDO',  # noqa: E501
+    evm_address_to_identifier('0x2be5e8c109e2197D077D13A82dAead6a9b3433C5', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'TOKAMAK',  # noqa: E501
```

Comment:
Does bitfinex have TOKAMAK? I don't see it in their website. I do see TON

Reply:
You are right. No need to add tokamak here. I recall that I was searching for TON added tokamak and then noticed it was TOKAMAK what I added